src/common/obj_bencher.cc

```diff
@@ -376,7 +376,17 @@ int ObjBencher::fetch_bench_metadata(const std::string& metadata_file,
   } else {
     *op_size = *object_size;
   }
-  unsigned ops_per_object = *object_size / *op_size;
+
+  unsigned ops_per_object;
+
```

Comment:
ditto.

Reply:
we don't fix the fix in the single PR. please squash this change into 8bd87fd40493571fe91d1aba0e1c7bc20a28e4f3 where this line is added.

Reply:
Yeah, I'll squash the commits

Reply:
All changes squashed into a single commit c28a00cfa5a67267243ce0c5a81d88fe1cb613e5